// _@vuepress_core@1.5.0@@vuepress/core/.temp/dynamic/vuepress_blog/paginations.js
/**
 * Generated by "@vuepress/plugin-blog"
 */

import sorters from './pageSorters'
import filters from './pageFilters'

export default [{
  pid: "tags",
  id: "Hello",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Hello/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "阿里云七天训练营",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/阿里云七天训练营/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "git/github",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/git/github/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "杂谈",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/杂谈/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "后端笔记",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/后端笔记/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "知识手册",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/知识手册/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
}]
