// _@vuepress_core@1.5.0@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "C:\\Users\\OeilShine\\Desktop\\OeilShine\\oeilshine.github.io\\node_modules\\_@vuepress_core@1.5.0@@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-b62f0354",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-b62f0354").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-28d66c58",
    path: "/blogs/category1/2018/121501.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-28d66c58").then(next)
    },
  },
  {
    name: "v-278deed0",
    path: "/blogs/category1/2019/092101.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-278deed0").then(next)
    },
  },
  {
    name: "v-ebd9ee90",
    path: "/blogs/category2/2016/121501.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ebd9ee90").then(next)
    },
  },
  {
    name: "v-4bb38ee5",
    path: "/blogs/mine/WeEngine.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4bb38ee5").then(next)
    },
  },
  {
    name: "v-4d75a4f8",
    path: "/blogs/category2/2017/092101.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4d75a4f8").then(next)
    },
  },
  {
    name: "v-5771f0e6",
    path: "/docs/theme-reco/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5771f0e6").then(next)
    },
  },
  {
    path: "/docs/theme-reco/index.html",
    redirect: "/docs/theme-reco/"
  },
  {
    name: "v-c9ca85b6",
    path: "/blogs/mine/MediaWiki.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c9ca85b6").then(next)
    },
  },
  {
    name: "v-ea100476",
    path: "/blogs/mine/mine.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ea100476").then(next)
    },
  },
  {
    name: "v-ef22f136",
    path: "/docs/theme-reco/api.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ef22f136").then(next)
    },
  },
  {
    name: "v-75522a8a",
    path: "/docs/theme-reco/plugin.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-75522a8a").then(next)
    },
  },
  {
    name: "v-304ae6f6",
    path: "/docs/theme-reco/theme.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-304ae6f6").then(next)
    },
  },
  {
    name: "v-2f65a16a",
    path: "/blogs/other/guide.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2f65a16a").then(next)
    },
  },
  {
    name: "v-b1564aac",
    path: "/tag/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tags", "v-b1564aac").then(next)
    },
    meta: {"pid":"tags","id":"tags"}
  },
  {
    path: "/tag/index.html",
    redirect: "/tag/"
  },
  {
    name: "v-ef9325c4",
    path: "/categories/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("FrontmatterKey", "v-ef9325c4").then(next)
    },
    meta: {"pid":"categories","id":"categories"}
  },
  {
    path: "/categories/index.html",
    redirect: "/categories/"
  },
  {
    name: "v-6319eb4e",
    path: "/timeline/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("TimeLines", "v-6319eb4e").then(next)
    },
    meta: {"pid":"timeline","id":"timeline"}
  },
  {
    path: "/timeline/index.html",
    redirect: "/timeline/"
  },
  {
    name: "v-1368f264",
    path: "/tag/tag1/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-1368f264").then(next)
    },
    meta: {"pid":"tags","id":"tag1"}
  },
  {
    path: "/tag/tag1/index.html",
    redirect: "/tag/tag1/"
  },
  {
    name: "v-1368f226",
    path: "/tag/tag2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-1368f226").then(next)
    },
    meta: {"pid":"tags","id":"tag2"}
  },
  {
    path: "/tag/tag2/index.html",
    redirect: "/tag/tag2/"
  },
  {
    name: "v-1368f1e8",
    path: "/tag/tag3/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-1368f1e8").then(next)
    },
    meta: {"pid":"tags","id":"tag3"}
  },
  {
    path: "/tag/tag3/index.html",
    redirect: "/tag/tag3/"
  },
  {
    name: "v-1368f1aa",
    path: "/tag/tag4/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-1368f1aa").then(next)
    },
    meta: {"pid":"tags","id":"tag4"}
  },
  {
    path: "/tag/tag4/index.html",
    redirect: "/tag/tag4/"
  },
  {
    name: "v-5ce69b3a",
    path: "/categories/category1/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-5ce69b3a").then(next)
    },
    meta: {"pid":"categories","id":"category1"}
  },
  {
    path: "/categories/category1/index.html",
    redirect: "/categories/category1/"
  },
  {
    name: "v-5ce69b59",
    path: "/categories/category2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-5ce69b59").then(next)
    },
    meta: {"pid":"categories","id":"category2"}
  },
  {
    path: "/categories/category2/index.html",
    redirect: "/categories/category2/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
