// _@vuepress_core@1.5.0@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "C:\\Users\\OeilShine\\Desktop\\OeilShine\\oeilshine.github.io\\node_modules\\_@vuepress_core@1.5.0@@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-b62f0354",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-b62f0354").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-c834e5d0",
    path: "/blogs/diary/2020/mine.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c834e5d0").then(next)
    },
  },
  {
    name: "v-2f65a16a",
    path: "/blogs/other/guide.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2f65a16a").then(next)
    },
  },
  {
    name: "v-1c1df235",
    path: "/blogs/server/2020/aliyun/MediaWiki.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1c1df235").then(next)
    },
  },
  {
    name: "v-04098658",
    path: "/blogs/handbook/2020/git.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-04098658").then(next)
    },
  },
  {
    name: "v-59a7e4f6",
    path: "/blogs/server/2020/aliyun/WeEngine.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-59a7e4f6").then(next)
    },
  },
  {
    name: "v-5771f0e6",
    path: "/docs/theme-reco/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5771f0e6").then(next)
    },
  },
  {
    path: "/docs/theme-reco/index.html",
    redirect: "/docs/theme-reco/"
  },
  {
    name: "v-ef22f136",
    path: "/docs/theme-reco/api.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ef22f136").then(next)
    },
  },
  {
    name: "v-75522a8a",
    path: "/docs/theme-reco/plugin.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-75522a8a").then(next)
    },
  },
  {
    name: "v-304ae6f6",
    path: "/docs/theme-reco/theme.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-304ae6f6").then(next)
    },
  },
  {
    name: "v-b1564aac",
    path: "/tag/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tags", "v-b1564aac").then(next)
    },
    meta: {"pid":"tags","id":"tags"}
  },
  {
    path: "/tag/index.html",
    redirect: "/tag/"
  },
  {
    name: "v-ef9325c4",
    path: "/categories/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("FrontmatterKey", "v-ef9325c4").then(next)
    },
    meta: {"pid":"categories","id":"categories"}
  },
  {
    path: "/categories/index.html",
    redirect: "/categories/"
  },
  {
    name: "v-6319eb4e",
    path: "/timeline/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("TimeLines", "v-6319eb4e").then(next)
    },
    meta: {"pid":"timeline","id":"timeline"}
  },
  {
    path: "/timeline/index.html",
    redirect: "/timeline/"
  },
  {
    name: "v-ef68be32",
    path: "/tag/Hello/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-ef68be32").then(next)
    },
    meta: {"pid":"tags","id":"Hello"}
  },
  {
    path: "/tag/Hello/index.html",
    redirect: "/tag/Hello/"
  },
  {
    name: "v-79a203dc",
    path: "/tag/阿里云七天训练营/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-79a203dc").then(next)
    },
    meta: {"pid":"tags","id":"阿里云七天训练营"}
  },
  {
    path: "/tag/阿里云七天训练营/index.html",
    redirect: "/tag/阿里云七天训练营/"
  },
  {
    path: "/tag/%E9%98%BF%E9%87%8C%E4%BA%91%E4%B8%83%E5%A4%A9%E8%AE%AD%E7%BB%83%E8%90%A5/",
    redirect: "/tag/阿里云七天训练营/"
  },
  {
    name: "v-06541af6",
    path: "/tag/git/github/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-06541af6").then(next)
    },
    meta: {"pid":"tags","id":"git/github"}
  },
  {
    path: "/tag/git/github/index.html",
    redirect: "/tag/git/github/"
  },
  {
    name: "v-644c5aac",
    path: "/categories/杂谈/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-644c5aac").then(next)
    },
    meta: {"pid":"categories","id":"杂谈"}
  },
  {
    path: "/categories/杂谈/index.html",
    redirect: "/categories/杂谈/"
  },
  {
    path: "/categories/%E6%9D%82%E8%B0%88/",
    redirect: "/categories/杂谈/"
  },
  {
    name: "v-2da2804f",
    path: "/categories/后端笔记/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-2da2804f").then(next)
    },
    meta: {"pid":"categories","id":"后端笔记"}
  },
  {
    path: "/categories/后端笔记/index.html",
    redirect: "/categories/后端笔记/"
  },
  {
    path: "/categories/%E5%90%8E%E7%AB%AF%E7%AC%94%E8%AE%B0/",
    redirect: "/categories/后端笔记/"
  },
  {
    name: "v-2ebc264a",
    path: "/categories/知识手册/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-2ebc264a").then(next)
    },
    meta: {"pid":"categories","id":"知识手册"}
  },
  {
    path: "/categories/知识手册/index.html",
    redirect: "/categories/知识手册/"
  },
  {
    path: "/categories/%E7%9F%A5%E8%AF%86%E6%89%8B%E5%86%8C/",
    redirect: "/categories/知识手册/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
